integrationTests/frontend/staking/staking_test.go

```diff
@@ -186,20 +186,64 @@ func TestValidatorToDelegationManagerWithNewContract(t *testing.T) {
 	nonce++
 
 	///////////------- send stake tx and check sender's balance
-	var txData string
 	genesisBlock := nodes[0].GenesisBlocks[core.MetachainShardId]
 	metaBlock := genesisBlock.(*block.MetaBlock)
 	nodePrice := big.NewInt(0).Set(metaBlock.EpochStart.Economics.NodePrice)
-	oneEncoded := hex.EncodeToString(big.NewInt(1).Bytes())
 
 	log.Info("using tx sign pk for staking", "pk", pkString)
 
 	frontendBLSPubkey, err := hex.DecodeString("309befb6387288380edda61ce174b12d42ad161d19361dfcf7e61e6a4e812caf07e45a5a1c5c1e6e1f2f4d84d794dc16d9c9db0088397d85002194b773c30a8b7839324b3b80d9b8510fe53385ba7b7383c96a4c07810db31d84b0feefafbd03")
 	require.Nil(t, err)
 	frontendHexSignature := "17b1f945404c0c98d2e69a576f3635f4ebe77cd396561566afb969333b0da053e7485b61ef10311f512e3ec2f351ee95"
 
+	nonce, round = generateSendAndWaitToExecuteStakeTransaction(
+		t,
+		nodes,
+		stakingWalletAccount,
+		idxProposers,
+		nodePrice,
+		frontendBLSPubkey,
+		frontendHexSignature,
+		nonce,
+		round,
+	)
+
+	time.Sleep(time.Second)
+
+	testStakingWasDone(t, nodes, frontendBLSPubkey)
+
+	saveDelegationContractsList(nodes)
```

Comment:
you could move `saveDelegationContractsList` and `saveDelegationManagerConfig` in a common package and export it. 
`saveDelegationManagerConfig` is duplicated in `stakingSC_test`

Reply:
done